packages/core/src/interpreter.ts

```diff
@@ -570,10 +580,6 @@ export function createActor<TMachine extends AnyStateMachine>(
     : MissingImplementationsError<TMachine['__TResolvedTypesMeta']>,
   options?: ActorOptions<TMachine>
 ): Actor<TMachine>;
-export function createActor<TLogic extends AnyActorLogic>(
-  logic: TLogic,
-  options?: ActorOptions<TLogic>
-): Actor<TLogic>;
```

Comment:
We might still need this, since machines are handled differently.

Reply:
Wasn’t removed, but was moved up. 

Moved because I noticed IntelliSense will suggest the first function override it finds, and I thought we'd want the actor logic one to be listed first in the list.

<img width="646" src="https://github.com/statelyai/xstate/assets/23459/7424e615-91ef-4c5e-b414-af2ec84b5593">